pysat/_instrument.py

```diff
@@ -1053,32 +1053,34 @@ def _assign_funcs(self, by_name=False, inst_module=None):
             # Either flags are not specified, or this combo is not
             self._password_req = False
 
-    def __str__(self):
-
-        output_str = '\npysat Instrument object\n'
+    def __repr__(self):
+        # Print the basic Instrument properties
+        output_str = 'pysat Instrument object\n'
         output_str += '-----------------------\n'
         output_str += 'Platform: ' + self.platform + '\n'
         output_str += 'Name: ' + self.name + '\n'
         output_str += 'Tag: ' + self.tag + '\n'
         output_str += 'Satellite id: ' + self.sat_id + '\n'
 
+        return output_str
+
+    def __str__(self):
+        # Get the basic Instrument properties
+        output_str = self.__repr__()
+
+        # Print out the data processing information
         output_str += '\nData Processing\n'
         output_str += '---------------\n'
         output_str += 'Cleaning Level: ' + self.clean_level + '\n'
         output_str += 'Data Padding: ' + self.pad.__repr__() + '\n'
         output_str += 'Keyword Arguments Passed to load(): '
-        output_str += self.kwargs.__repr__() + '\nCustom Functions : \n'
-        if len(self.custom._functions) > 0:
-            for func in self.custom._functions:
-                output_str += '    ' + func.__repr__() + '\n'
-        else:
-            output_str += '    ' + 'No functions applied.\n'
+        output_str += self.kwargs.__repr__() + '\n'
```

Comment:
```
pysat.Instrument('pysat', 'testing', sat_id='', tag='')
Data Processing
---------------
Cleaning Level: clean
Data Padding: None
Keyword Arguments Passed to load(): {'sim_multi_file_right': False, 'sim_multi_file_left': False, 'root_date': None, 'file_date_range': None, 'malformed_index': False, 'mangle_file_dates': False}
Custom Functions: 0 applied
Local File Statistics
---------------------
Number of files: 1096
Date Range: 01 January 2008 --- 31 December 2010

Loaded Data Statistics
----------------------
No loaded data.

```

It would be nice if the keyword arguments had some additional formatting.

Reply:
line breaks in between sections?

Reply:
I agree about the line break between sections, but not to more formatting for the keyword arguments.  I don't really see the value in printing these out here (most of them are default and not user-specified), and would prefer that they were removed.

Reply:
Ok, I decided the load kwargs are ok, but I think the current formatting is fine.